solr/server/solr/solr.xml

```diff
@@ -30,6 +30,7 @@
 
   <int name="maxBooleanClauses">${solr.max.booleanClauses:1024}</int>
   <str name="sharedLib">${solr.sharedLib:}</str>
+  <str name="modules">${solr.modules:}</str>
```

Comment:
I think this should go in the ref guide's page on the solr.xml

Reply:
I was sure I did that, but now I have committed docs in `configuring-solr-xml.adoc`